rpcserver.go

```diff
@@ -2394,8 +2394,7 @@ func (r *rpcServer) sendPayment(stream *paymentStream) error {
 				} else if err != nil {
 					select {
 					case errChan <- err:
-					case <-reqQuit:
-						return
+					default:
```

Comment:
should we log the error instead of just dropping it in the default case? agree that the first error is all that matters in returning to caller, but would be nice to see if other things are failing as well

Reply:
Good idea! Decided to log in both cases, as I imagine it is useful to see these error on the grpc server as well.